### ISSUE TITLE
Double Constants

### ISSUE THREAD

**Issue by trans** [LogosInitiative/logla#4]
Allowing double constants would make borrowings and compound words easier to form, for example `fimma`. To distinguish these words without ambiguity, the pronunciation of the double constant can be aspirated and ideally held for a period perceptibly longer than normal. So `fimma` would sound like <tt>fim<sup>h</sup>a</tt>. 

I do not believe this would cause any conflicts with current morphology because it places the `h` sound such that it is preceded by a constant instead of a vowel.

**Comment by thetommoore** [comment]
Another way to do this would be in the Japanese fashion, adding the full syllabic form of the consonant before it. For example, `fimma` would be /fi.m.ma/. With your proposal, it would probably make more sense to just call them aspirated consonants and write `fim'a`.

**Comment by trans** [comment]
Good point about the aspiration. My goal in this, by the way, is to make it easier to form compound words from roots (i.e. lujvo from rafsi).

Another alternative is gemination, i.e. fimːa. But I never much cared for the idea of long constants. Not sure. Maybe I could get used to it.

**Comment by thetommoore** [comment]
Long consonants sound pretty good in Japanese IMO, so you might want to look into that. It has more to do with the moraic prosody, I think.
